base-template.sh

```diff
@@ -1000,7 +941,7 @@ read_single_repo_information() {
 #   0 if they were, 1 otherwise
 #####################################################
 is_single_repo() {
-    [ "$IS_SINGLE_REPO" = "yes" ] || return 1
+    [ "$(git config --get --local githooks.single.install)" = "yes" ] || return 1
```

Comment:
Should this also deal with `true` ?

Reply:
Forgot that, and dont need to change this because we any way gonna remove it.